toolkits/springernature/packages/springernature-forms/scss/50-components/_forms.scss

```diff
@@ -0,0 +1,121 @@
+.c-form__section {
```

Comment:
class name format should follow the component name, so in this case rather than `c-form` it should be `c-forms`